docs/application/vscode-ext/web.md

```diff
@@ -333,46 +333,106 @@ To debug application, follow these steps:
 
     ![Debugging you application in the Web Simulator](media/tizen_web18.png)
 
-### Run Design Editor
+### Quick Action Execution
+Right-click on the **project folder** to use the extension function in the menu.
+
+For more information on each function, see:
+- [Tizen Web: Build Tizen App](#Build-Project)
+- [Tizen Web: Debug Tizen App](#Deploy-and-Run-Application-in-Emulator)
+- [Tizen Web: Install Tizen App](#Deploy-and-Run-Application-in-Emulator)
+- [Tizen Web: Make Exclude Build List](#Exclude-Build-List)
+- [Tizen Web: Run Tizen App](#Deploy-and-Run-Application-in-Emulator)
+
+![Design Editor launch](media/tizen_web34.png)
+
 
-To run Design Editor, follow one of the below ways:
+## Tizen Web with Design Editor
 
-> **Note** 
-> 
-> You must select the **HTML extension file** in the file explorer.
+Visual Studio Code Extension for Tizen Web supports WYSIWYG Design Editor features such as **Preview**, **JavaScript Assistant**, **Structure View**, **New Page**, **HTML Assistant** , **Undo** , and **Redo**.
+You must select the **HTML extension file** in the file explorer before you launch Design Editor.
 
-- Command Palette
-    1. In VS Code, click **View** > **Command Palette**.
-    2. In the input field of the **Command Palette** that appears, type **Tizen Web: Run Design-Editor** and press **Enter**.
+### Run Design Editor
+
+To run Design Editor, perform one of the following:
 
-        A new tab is created and the Design Editor is launched.
+- Using Command Palette
+    1. To launch Design Editor, in VS Code, click **View** > **Command Palette**.
```

Comment:
@Reni-90 The requirement for the consistency is not needed to the case like this. The "consistency" doesn't mean "use same structure of words to everywhere ignoring the sentence flow".
You don't need to strictly apply" in VS Code"at here, because it breaks the sentence to many flows. (Also "in VS Code" itself is not a thing that needs consistency)

You could have suggested "To launch Design Editor, click **View** > **Command Palette** in the VS Code." instead.

Reply:
Which is correct ??
To launch Design Editor, click View > Command Palette in the VS Code.
or 
To launch Design Editor, in VS Code, click **View** > **Command Palette**.

Reply:
@mijin85cho 
If consistency is not required, we can revert back to the original sentence. As I was going through other sentences in the same file, I suggested the change. 
Thank you for explaining.

@singa2000 
Please change the sentence to:
 "To launch Design Editor, click **View > Command Palette** in VS Code."

Reply:
Sure. I changed